### PR TITLE
python3Packages.shapely_1_8: disable failing test

### PR DESCRIPTION
**Pull request by imincik**
## Description of changes

Disable failing test to [unbreak kikit](https://github.com/NixOS/nixpkgs/issues/325220).

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/shapely/1.8.nix [hunk 1]
```diff
@@ -64,16 +64,21 @@ buildPythonPackage rec {
...
-    # It is apparently some sandbox or no-sandbox related thing on macOS only
-    # though.
-    "test_error_handler_exception"
-    "test_error_handler"
-    "test_info_handler"
-  ];
+  disabledTests =
+    [
+      # started to fail after geos update from 3.11.2 to 3.11.4
+      "test_parallel_offset_linestring"
```
Comment (l0b0) [inline pkgs/development/python-modules/shapely/1.8.nix:70]: Is there an upstream issue, so that we can eventually remove this line?

Comment (imincik) [reply pkgs/development/python-modules/shapely/1.8.nix:70]: shapely 1.8 is old version. I guess it is out of support.

**Review by jfly — COMMENTED** [review]
@imincik, sorry for the delayed review.

It looks like kikit has started allowing for newer versions of shapely (https://github.com/yaqwsx/KiKit/issues/574#issuecomment-1879721181)!

How about we instead change kikit to use shapely 2 and drop shapely 1.8 from nixpkgs? Basically, what you tried to do over in https://github.com/NixOS/nixpkgs/pull/268277 before @matusf and I stopped you (sorry!).

**Comment by imincik** [comment]
> How about we instead change kikit to use shapely 2 and drop shapely 1.8 from nixpkgs?

I am only happy with that.

**Comment by jfly** [comment]
@imincik, unfortunately that turned into a minor adventure. I've dropped shapely 1.8 in https://github.com/NixOS/nixpkgs/pull/336124.

I'd suggest that we close this PR in favor of that one.

**Comment by imincik** [comment]
> @imincik, unfortunately that turned into a minor adventure. I've dropped shapely 1.8 in #336124.
> 
> I'd suggest that we close this PR in favor of that one.

OK, I agree.

**Pull request closed without merging**